samples/samples-js/AddProduct/function.json

```diff
@@ -0,0 +1,27 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "name": "req",
+      "direction": "in",
+      "type": "httpTrigger",
+      "methods": [
+        "get",
+        "post"
+      ]
+    },
+    {
+      "name": "$return",
+      "type": "http",
+      "direction": "out"
+    },
+    {
+      "name": "product",
+      "type": "sql",
+      "direction": "out",
+      "commandText": "[TestDB].[dbo].[Products]",
```

Comment:
I needed to change this locally. Do we hardcode TestDB anywhere else?

Reply:
good catch, yeah it was my dbname, removed it!